mdata/cache/cache_mock.go

```diff
@@ -12,9 +14,17 @@ type MockCache struct {
 	CacheIfHotCb    func()
 	StopCount       int
 	SearchCount     int
+	DelMetricRes    CCDelMetricResult
+	DelMetricKeys   []string
+}
+
+func NewMockCache() *MockCache {
+	return &MockCache{
+		DelMetricKeys: make([]string, 0),
```

Comment:
can just leave it at zero value

Reply:
ok, fixed https://github.com/grafana/metrictank/pull/555/commits/c90ad3f3c5f7824301122ee313bb0ad05fd4bfe6